server/stream.go

```diff
@@ -812,9 +812,9 @@ func (a *Account) addStreamWithAssignment(config *StreamConfig, fsConfig *FileSt
 
 	// Create our pubAck template here. Better than json marshal each time on success.
 	if domain := s.getOpts().JetStreamDomain; domain != _EMPTY_ {
-		mset.pubAck = []byte(fmt.Sprintf("{%q:%q, %q:%q, %q:", "stream", cfg.Name, "domain", domain, "seq"))
+		mset.pubAck = []byte(fmt.Sprintf("{%q:%q,%q:%q,%q:", "stream", cfg.Name, "domain", domain, "seq"))
```

Comment:
Realise this was existing code, but it's probably better to use `fmt.Appendf()` instead of `[]byte(fmt.Sprintf())` here in both cases.